conmon/Makefile

```diff
@@ -1,12 +1,19 @@
+include ../Makefile.inc
+
 src = $(wildcard *.c)
 obj = $(src:.c=.o)
 
 override LIBS += $(shell pkg-config --libs glib-2.0)
 override CFLAGS += -std=c99 -Os -Wall -Wextra $(shell pkg-config --cflags glib-2.0)
 
-conmon: $(obj)
+conmon: $(obj) version.h
```

Comment:
This `version.h` is a complile-time dependency not a link-time dependency:

```console
$ make clean
$ make
cc -std=c99 -Os -Wall -Wextra -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include   -c -o conmon.o conmon.c
conmon.c:29:21: fatal error: version.h: No such file or directory
 #include "version.h"
                     ^
compilation terminated.
make: *** [<builtin>: conmon.o] Error 1
```

Can we setup our own `conmon.o` prereqs?

```makefile
conmon: $(obj)
    $(CC) -o ../bin/$@ $^ $(CFLAGS) $(LIBS)

conmon.o: conmon.c version.h
```

or drop `version.h` and [append][2] `-DVERSION=$(VERSION) -DGIT_COMMIT=$(GIT_COMMIT) ` to `CFLAGS`:

```makefile
CFLAGS += -DVERSION=$(VERSION) -DGIT_COMMIT=$(GIT_COMMIT)
```
[1]: https://www.gnu.org/software/make/manual/html_node/Catalogue-of-Rules.html#index-cc
[2]: https://www.gnu.org/software/make/manual/html_node/Appending.html#index-_002b_003d

Reply:
Using `-D` to eliminate the need for `version.h` seems like the more intuitive of those options